dev/generate_bom.py

```diff
@@ -155,9 +155,15 @@ def write_docker_version_files(self):
     """
     for comp in self.__component_versions:
       gradle_version = self.__version_from_tag(comp)
-      docker_tag = '{reg}/{comp}:{tag}'.format(reg=self.__docker_registry,
-                                               comp=comp,
-                                               tag=gradle_version)
+      docker_tag = ''
+      if comp == 'spinnaker-monitoring':
+        docker_tag = '{reg}/{comp}-daemon:{tag}'.format(reg=self.__docker_registry,
+                                                        comp=comp,
```

Comment:
make a comment here (and in other file) indicating what makes spinnaker-monitoring special is that the deb name is different from the repo name (because it has multiple debs).

I'm not sure why third-party isnt here. It does have the dashboards and other monitoring support.